### DISCUSSION TITLE
Is it fully open source? Why aab binary file is there ?

### DISCUSSION THREAD

**Discussion by ghost** [florisboard/florisboard#2380] [Q&A] [answered]
Binary file link: https://github.com/florisboard/florisboard/blob/master/app/beta/app-beta.aab

It seems like this binary file is beta version file. If i am right, why beta version is not fully open source.

**Comment by lm41** [comment, marked as answer]
This is the builded binary of the source. I don't know why it is there but the beta is completely open source.

**Reply by patrickgold** [reply]
Oh oops Android Studio changed the output path of binary aab files (output of compiled ifles for Google Play releases), and I accidentally committed it.

**Comment by ghost** [comment]
This file was added recently through this [release commit](https://github.com/florisboard/florisboard/commit/5bb7c6f7869cd7b39c2ed13009652baa98fe2404)

Hmm I think this file was added by mistake may be during building release apk
